api/src/utils/get-secret.ts

```diff
@@ -0,0 +1,26 @@
+import { useEnv } from '@directus/env';
+import { nanoid } from 'nanoid';
+import { useLogger } from '../logger.js';
+
+export const _cache: { secret: string | null } = { secret: null };
+
+export const getSecret = () => {
+	if (_cache.secret) {
+		return _cache.secret;
+	}
+
+	const env = useEnv();
+	const logger = useLogger();
+
+	if (env['SECRET']) {
+		return env['SECRET'] as string;
```

Comment:
Since we have it consolidated at this one point, should we explicitly cast?

```suggestion
		return String(env['SECRET']);
```